recipes/timemory/conda_build_config.yaml

```diff
@@ -0,0 +1,16 @@
+# required for clock_gettime_nsec_np, etc.
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - "10.12"                # [osx and x86_64]
```

Comment:
Could you check whether it is sufficient to only update the SDK and not the target?

Reply:
Ok, deployment target does not need to be 10.12 but I re-disabled OSX because it still fails with `PyMUTEX_LOCK(gil->mutex) failed`